multimedia/mlt/Portfile

```diff
@@ -22,11 +22,12 @@ long_description    MLT is an open source multimedia framework, designed and dev
                     This package contains the files necessary for running applications that use \
                     the MLT multimedia framework.
 homepage            http://www.mltframework.org/
-master_sites        sourceforge:project/mlt/mlt
+master_sites        https://github.com/mltframework/mlt/releases/download/v${version}/ \
```

Comment:
If we're now retrieving from GitHub, perhaps a switch to the GitHub portgroup would be in order?

Reply:
Yes, that is the preferred method.